content/docs/dvclive/get-started.md

```diff
@@ -3,164 +3,189 @@
 
-If you use one of the supported [ML Frameworks](/doc/dvclive/ml-frameworks), you
-can jump directly to its corresponding page.
+First of all, you need to add DVCLive to your Python code to start logging
+metrics:
 
-</admon>
-
-## Workflow
-
-### Initialize DVCLive
+<toggle>
+<tab title="Keras">
 
 ```python
-from dvclive import Live
-
-live = Live()
+# train.py
+from dvclive.keras import DvcLiveCallback
+...
+model.fit(
+  train_dataset, validation_data=validation_dataset,
+  callbacks=[DvcLiveCallback()])
 ```
 
-See [`Live()`](/doc/dvclive/api-reference/live) for details.
-
-### (Optionally) Log parameters
+</tab>
 
-<toggle>
-<tab title="Single parameter">
+<tab title="Hugging Face">
 
 ```python
-live.log_param("num_classes", 10)
+# train.py
+from dvclive.huggingface import DvcLiveCallback
+
+. . .
+
+ trainer = Trainer(
+    model, args,
+    train_dataset=train_data,
+    eval_dataset=eval_data,
+    tokenizer=tokenizer,
+    compute_metrics=compute_metrics,
+)
+trainer.add_callback(DvcLiveCallback())
+trainer.train()
 ```
 
-See `Live.log_param()`.
-
 </tab>
-<tab title="Multiple parameters">
+<tab title="Pytorch Lightning">
 
 ```python
-params = {
-    "num_classes": 10,
-    "metrics": ["accuracy", "mae"],
-    "optimizer": "adam"
-}
-live.log_params(params)
-```
+# train.py
+from dvclive.lightning import DvcLiveLogger
 
-See `Live.log_params()`.
+. . .
+dvclive_logger = DvcLiveLogger()
+
+trainer = Trainer(logger=dvclive_logger)
+trainer.fit(model)
+```
 
 </tab>
 </toggle>
 
-### Log data
-
-<toggle>
-<tab title="Scalars">
+For other supported frameworks, you can find example snippets and API reference
+on the corresponding page of [ML Frameworks](/doc/dvclive/ml-frameworks).
 
-```python
-live.log("acc", 0.9)
-```
+If you prefer to use the DVCLive API directly, check the
+[API Reference](/doc/dvclive/api-reference).
 
-See `Live.log()`.
+### Run the script
 
-</tab>
-<tab title="Images">
+Once you have added DVCLive to our python code, you can run the script as you
+would usually do:
 
-```python
-img = np.ones((500, 500, 3), np.uint8)
-live.log_image("image.png", img)
+```dvc
+$ python train.py
 ```
 
-See `Live.log_image()`.
+DVCLive will generate a
+[metrics report](/doc/dvclive/api-reference/live/make_report) in
+`dvclive/report.html` containing all the logged data. It will be automatically
+updated during training on each `step` update:
 
-</tab>
-<tab title="Plots">
+![HTML report](/img/dvclive-html.gif).
 
-```python
-y_true = [0, 0, 1, 1]
-y_pred = [0.2, 0.5, 0.3, 0.8]
-live.log_plot("roc", y_true, y_score)
-```
+You can find more details about the structure of files generated by DVCLive in
+the [Output folder structure](/doc/dvclive/outputs).
 
-See `Live.log_plot()`.
+## Set up DVC
 
-</tab>
-</toggle>
+To build on the previous section, you can optionally wrap our code with DVC to
+run [DVC experiments](/doc/user-guide/experiment-management/) and enable new
+ways to compare and visualize our experiments.
 
```

Comment:
Hopefully, we can improve this section soon. For now, what do you about making the entire setup an expandable section? Realistically, right now a new user probably needs to go the DVC docs and get set up with DVC before coming back to DVCLive, so I don't think it's worth pushing down the other content to explain how to setup DVC.

Reply:
I still want to highlight the `dvc.yaml` syntax required (`metrics`, `plots`) to get the things rendered. 
Looking at it right now I don't really like that the `dvc.yaml` is hidden in an expandable section.

Reply:
I removed `dvc init` and `exp init` but still displayed the `dvc.yaml`. Still think it makes sense as minimum for building the sections below about different visualization